README.md

```diff
@@ -1,69 +1,97 @@
 # Merge Confidence
 
-Merge Confidence identifies and flags undeclared breaking releases based on analysis of test and release adoption data across WhiteSource Renovate’s early-adopting user base.
-The new feature was created to help users avoid the pain of un-mergeable Pull Requests or worse -- a broken dependency in production.
+Look at the Merge Confidence badges before merging to:
```

Comment:
Let's put the important stuff for the reader up first, and sell them on this awesome feature. 😄